UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -119,7 +119,11 @@ public async Task<string> CreateExchangeSet(SalesCatalogueServiceResponseQueueMe
 
         public async Task<string> CreateLargeExchangeSet(SalesCatalogueServiceResponseQueueMessage message, string currentUtcDate, string largeExchangeSetFolderName)
         {
+            DateTime createExchangeSetTaskStartedAt = DateTime.UtcNow;
+            DateTime createExchangeSetTaskCompletedAt;
```

Comment:
Redundant variable - just pass DateTime.UtcNow in the parameter rather than setting before